src/dynamodb_encryption_sdk/structures.py

```diff
@@ -304,7 +304,7 @@ class TableInfo(object):
         default=None
     )
     _secondary_indexes = attr.ib(
-        validator=attr.validators.optional(iterable_validator(set, TableIndex)),
+        validator=attr.validators.optional(iterable_validator(list, TableIndex)),
```

Comment:
What about the type annotation for `secondary_indexes` on L#341? Did that intentionally not change?

Reply:
nope, that was an oversight